romancal/pipeline/highlevel_pipeline.py

```diff
@@ -55,13 +56,16 @@ def process(self, input):
             return
 
         if file_type == "asn":
+            input = ModelContainer(input)
+            self.output_file = input.asn_table["products"][0]["name"]
             self.skymatch.suffix = "skymatch"
             result = self.skymatch(input)
-            self.skymatch.suffix = "outlier_detection"
-            # result = self.outlier_detection(input)
-            self.skymatch.suffix = "i2d"
+            self.outlier_detection.suffix = "outlier_detection"
+            result = self.outlier_detection(input)
+            self.resample.suffix = "i2d"
             result = self.resample(input)
```

Comment:
Same comment about the input to resample.

Reply:
Fixed